openfisca_aotearoa/tests/parental_leave/parental_leave.yaml

```diff
@@ -0,0 +1,24 @@
+---
+
+- name: "Paid Parental Leave"
```

Comment:
Thanks for the mahi, I'd agree we need more tests and unique descriptive names for what each test is testing.